nimbus.fml.yaml

```diff
@@ -189,10 +189,10 @@ features:
         type: Map<MR2022Section, Boolean>
         default:
           {
-            "home-onboarding-dialog-existing-users": false,
-            "sync-cfr": false,
-            "wallpapers-selection-tool": false,
-            "jump-back-in-cfr": false,
+            "home-onboarding-dialog-existing-users": true,
+            "sync-cfr": true,
+            "wallpapers-selection-tool": true,
+            "jump-back-in-cfr": true,
```

Comment:
1. Did we intend to make this default true for all channels? 
2. Are we also no longer running a nimbus experiment since the defaults are all true?

Reply:
> Did we intend to make this default true for all channels?

Yes.

> Are we also no longer running a nimbus experiment since the defaults are all true?

Yes, we are but we want all the users get the enabled experience by default, on what we are experimenting is on disabling the features for some users.